ocs_ci/ocs/ocp.py

```diff
@@ -441,8 +441,9 @@ def new_project(self, project_name, policy=constants.PSA_BASELINE):
         """
         ocp = OCP(kind="namespace")
         command = f"oc new-project {project_name}"
-        if f'Now using project "{project_name}"' in run_cmd(
-            f"{command}", threading_lock=self.threading_lock
+        cmd_out = run_cmd(f"{command}", threading_lock=self.threading_lock)
+        if (f'Now using project "{project_name}"' in cmd_out) or (
+            f'Already on project "{project_name}"' in cmd_out
```

Comment:
If you already on project "project_name" then why you are creating again same project?

Reply:
@avd-sagare refer the issue https://github.com/red-hat-storage/ocs-ci/issues/6512